docs/migrations/web/MIGRATION-v3.md

```diff
@@ -6,12 +6,173 @@ Introducing version 3 of the _spirit-web_ package.
+| ----- | -------------- | -------------- |
+| 0     | space-0        | space-0        |
+| 1 px  | space-100      | space-100      |
+| 2 px  | space-200      | space-200      |
+| 4 px  | space-300      | space-300      |
+| 6 px  | —              | space-400      |
+| 8 px  | space-400      | space-500      |
+| 12 px | space-500      | space-600      |
+| 16 px | space-600      | space-700      |
+| 20 px | —              | space-800      |
+| 24 px | space-700      | space-900      |
+| 32 px | space-800      | space-1000     |
+| 40 px | space-900      | space-1100     |
+| 48 px | space-1000     | space-1200     |
+| 56 px | —              | space-1300     |
+| 64 px | space-1100     | space-1400     |
+| 72 px | —              | space-1500     |
+| 80 px | space-1200     | space-1600     |
+| 96 px | —              | space-1700     |
+
+If you are going to use fulltext search-and-replace in your project, we suggest you to
+start from the largest values and work your way down to the smallest ones.
+This way you will avoid replacing values you already replaced.
+
+### Helpers: Typography and Headings
+
+As the word `text` was omitted from the typography tokens, the typography helpers
+were updated to reflect this change. Also, the `heading` helpers were updated to
+support the emphasis.
+
+#### Migration Guide
+
+Remove the `text` infix from `body` typography helpers.
+
+For example:
+
+- `.typography-body-large-text-bold` → `.typography-body-large-bold`
+- `.typography-body-xsmall-text-regular` → `.typography-body-xsmall-regular`
+
+More font weights are now available for headings.
+Rename `text` to `bold` in the `heading` helpers.
+
+For example:
+
+- `.typography-heading-xlarge-text` → `.typography-heading-xlarge-bold`
+- `.typography-heading-xsmall-text` → `.typography-heading-xsmall-bold`
+
+### Utilities: Removed and New Classes
+
+Some utility classes were removed and new ones were added.
+
+Removed utility classes:
+
+- `.text-brand-primary`
+- `.text-brand-secondary`
+- `.bg-basic`
+- `.bg-cover`
+- `.bg-inverted`
+
+New utility classes:
+
+- `.text-tertiary`
+- `.bg-primary`
+- `.bg-secondary`
+- `.bg-tertiary`
+
+### Migrating Your Own Components
+
+First, you need to update your design tokens. You can either use Supernova to export
+your design tokens in new structure, or you can manually update your tokens, or update
+our tokens package if you are using it.
+
+Here are the steps to migrate your own components to use the new design tokens and themes:
+
+1. If you load `foundation`, `components`, `helpers`, and `utilities` separately, you need
+   to `@forward 'themes'` before them in your main SCSS file. If you load the whole `scss` folder,
+   you don't need to do anything.
+2. Replace module load `@use '@tokens' as tokens;` in your components with `@use '@global' as global-tokens;`.
+3. Rename all the tokens in your components to use the new structure.
+   1. Replace all module name `tokens.*` → `global-tokens.*`.
```

Comment:
I didn't notice this before. What does it mean? 🤔

Reply:
updated